### PR TITLE
feat: add standalone model for generic-example

### PR DESCRIPTION
**Pull request by jkbmrz**
Adding a shell script to run the basic-example in the standalone mode.

### CODE REVIEW

File: gen3/neural-networks/generic-example/main_standalone.sh [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+fi
+
+if [ -n "$FPS_LIMIT" ]; then
+    sed -i "s|<FPS>|--fps_limit $FPS_LIMIT|g" "$OAKAPP"
+else
+    sed -i "s|<FPS>||g" "$OAKAPP"
+fi
+
+if [ -n "$MEDIA" ]; then
+    sed -i "s|<Media>|--media $MEDIA|g" "$OAKAPP"
```
[the remark this answers is not in the document]
Comment (jkbmrz) [reply gen3/neural-networks/generic-example/main_standalone.sh:48]: It works. The video gets moved to device automatically if present in the root folder. Will add a disclaimer!

File: gen3/neural-networks/generic-example/oakapp_template.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
-identifier = "com.luxonis.basic-example.device-decoding"
+identifier = "com.luxonis.generic-example.device-decoding"
```
[the remark this answers is not in the document]
Comment (jkbmrz) [reply gen3/neural-networks/generic-example/oakapp_template.toml]: Thanks!

File: gen3/neural-networks/generic-example/README.md
[the remark this answers is not in the document]
Comment (jkbmrz) [reply gen3/neural-networks/generic-example/README.md:1]: added in [5562391](https://github.com/luxonis/depthai-experiments/pull/564/commits/55623910de3cfa2ee454d5a9a2811f782ddb765a)

File: gen3/neural-networks/generic-example/main_standalone.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (jkbmrz) [reply gen3/neural-networks/generic-example/main_standalone.sh]: added in [5562391](https://github.com/luxonis/depthai-experiments/pull/564/commits/55623910de3cfa2ee454d5a9a2811f782ddb765a).

File: gen3/neural-networks/generic-example/README.md [hunk 3]
```diff
@@ -8,42 +8,72 @@ Moreover, you need to prepare a **Python 3.10** environment with the following p
...
+
+## Standalone Mode
+Running the example in the [Standalone mode](https://rvc4.docs.luxonis.com/software/depthai/standalone/), app runs entirely on the device.
+To run the example in this mode, first install the [oakctl](https://rvc4.docs.luxonis.com/software/tools/oakctl/) command-line tool (enables host-device interaction) as:
+```bash
+bash -c "$(curl -fsSL https://oakctl-releases.luxonis.com/oakctl-installer.sh)"
+```
+and run the example using the `main_standalone.sh` script:
+```bash
+bash main_standalone.sh \
```
Comment (klemen1999) [inline gen3/neural-networks/generic-example/README.md:62]: Let's rename to `run_standalone.sh` or if maybe have better name for it

**Review by klemen1999 — APPROVED** [review]

File: gen3/neural-networks/generic-example/README.md (hunk 3 above)
Comment (jkbmrz) [reply gen3/neural-networks/generic-example/README.md]: Added in [45e4f16](https://github.com/luxonis/depthai-experiments/pull/564/commits/45e4f166dcc2b3095b228d978aa91cc63542ce28).

**Pull request merged**